### DISCUSSION TITLE
Dexcom one transmiter status UNKNOWN

### DISCUSSION THREAD

**Discussion by spindulel** [NightscoutFoundation/xDrip#3374] [Bug reports]
I changed a new dexcom one sensor and it worked for a day. However yesterday evening it stopped and xdrip is stuck in transmitter status - "unknown.". I've renewed newest version of xdrip, but it didn't help. 
My Android model is Redme Note 11S
Any ideas how to manage this error?
![Screenshot_2024-03-01-21-04-42-932_com eveningoutpost dexdrip](https://github.com/NightscoutFoundation/xDrip/assets/161881080/a22ac0ed-f575-485e-9973-61fc9bd63583)

**Comment by spindulel** [comment]


**Reply by Navid200** [reply]
What is this icon?
![309370739-a22ac0ed-f575-485e-9973-61fc9bd63583](https://github.com/NightscoutFoundation/xDrip/assets/51497406/69828ab7-e6fe-417a-ad55-f84897b76a73)

**Reply by spindulel** [reply]
It means "The status of the System"

**Reply by Navid200** [reply]
You mean that is not an app icon?

**Comment by spindulel** [comment]
This is an option from xDrip meniu "System status" which has two windows.  The printscreen is one of the windows of "System status". If we as the users face any problem, this is the main window where we can look for transmitter's or sensor's status.

**Reply by Navid200** [reply]
You have an app.  You can see its icon in your screenshot.  I have put a red rectangle around it to bring it to your attention.  
Here it is again.  I am attaching it here again.
![309377277-69828ab7-e6fe-417a-ad55-f84897b76a73](https://github.com/NightscoutFoundation/xDrip/assets/51497406/dd19743c-f559-41d9-90a7-a02c7c67268d)

**Reply by spindulel** [reply]
Are you kidding me? This is Instagram icon. 
It has nothing to do with my issue. 
Do you have any ideas how to solve Dexcom One transmittor issue in xDrip?

**Reply by Navid200** [reply]
No,

Hopefully, someone else does and can help you.

**Reply by spindulel** [reply]
I see. What do you propose then? Could you please recommend who I should contact?

**Reply by Navid200** [reply]
You say you installed a new transmitter the day before.  But, the status page you have posted shows 42 for transmitter days.
Can you explain that?

**Reply by spindulel** [reply]
Sorry for confusion. I started a new sensor the day before and it worked ok. But then it stopped and I got this message that transmitter status iš UNKNOWN.

**Reply by Navid200** [reply]
Then, you should edit your first post and correct that to avoid others coming to incorrect conclusions.

You should also always include in the very first post (you can edit now) to include your phone brand and model for a connectivity issue.

Are all settings correct?
https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html

**Reply by spindulel** [reply]
Thank you! I edited my first post. Yes, settings seems corect.

**Reply by Navid200** [reply]
Do you have a Dexcom receiver to test to see if the transmitter works with that?
Do you have a phone that is on the Dexcom list of approved phones for the official Dexcom One app?

**Reply by spindulel** [reply]
No, I don't have it unfortunately. 
I use the same phone for receiving data for longer than a year and it is the first time I face such issues.

**Reply by Navid200** [reply]
What does the status page show now?

**Reply by spindulel** [reply]
It is the same.
![Screenshot_2024-03-02-21-39-51-973_com eveningoutpost dexdrip](https://github.com/NightscoutFoundation/xDrip/assets/161881080/86e2210a-4bb1-44c6-ae2c-6ff82809674c)

**Reply by Navid200** [reply]
Xiaomi phones are known for aggressively disabling apps that run in the background.
https://dontkillmyapp.com/xiaomi

You can use another phone to see if the problem is the same or not.

If it is, it may be the transmitter.
The Dexcom One transmitters are known to break a lot more than G6 transmitters.
If you can be sure that it is the transmitter, you can contact Dexcom.
If you had a receiver or if you had a phone that supported the Dexcom app and neither of those worked either, you could be sure that it was the transmitter.

**Reply by spindulel** [reply]
Hello again,
 I have tried to connect the transmitter with another phone - Samsung A54. And I get an error message "No valid timestamp stored for transmitter: C2DWLF"
Do you know what does that mean?
I also send a screen shoot of System status display (attached). 
![Uploading IMG_20240303_213843.jpg…]()

**Reply by Navid200** [reply]
When you are editing your post, there is a paper clip symbol that represents attachment.  If you use that, you should be able to attach an image.

Since I cannot see the image, I wonder if you have readings or not.  It would help if you show the status page instead of the logs.

**Reply by spindulel** [reply]
Here is the image 
![Screenshot_2024-03-04-16-34-44-192_com miui gallery](https://github.com/NightscoutFoundation/xDrip/assets/161881080/c65043e8-68ef-41e1-9858-29882c6cb78f)

**Reply by Navid200** [reply]
The transmitter could be dead.
Do you see the transmitter ping in a scanner?
https://navid200.github.io/xDrip/docs/Bluetooth-Scanner.html

**Reply by spindulel** [reply]
No, I don't see it. 
I've contacted local Dexcom support and they said that there were incresed number of xDrip users who faced with technical issues recently. Do you perhaps noticed such increse in your support line as well?
 I also see that last release was on 24th of February. Do you maybe know if there will be any updates of  in nearly future?

**Reply by Navid200** [reply]
No, there is no xDrip update in the works to address a malfunctioning transmitter.

**Reply by spindulel** [reply]
Yes, I understand that there is no possibility to eliminate the spoilage of transmitter. However maybe there are any possibilities to prevent future spoilages, if it is somehow connected to xDrip app. I know that there can be many other reasons as well, of cause.

**Reply by Navid200** [reply]
Can you perform an experiment?

1- Enable engineering mode: https://navid200.github.io/xDrip/docs/Engineering-Mode.html

2- In xDrip, change the following to 1.
Settings &#8722;> G5/G6/Dex1 Debug Settings &#8722;> Manual Slot Number 

Does it connect now?

**Reply by spindulel** [reply]
Yes, I could try to experiment. Could you please tell me what is "Manual Slot Number"? It is not transmitter number, right?

**Reply by Navid200** [reply]
No, it is the slot number.

**Reply by spindulel** [reply]
I see. What is slot number? Where can I find it? I've never needed such.

**Reply by Navid200** [reply]
I've already explained everything.  This was supposed to be an experiment.
You can forget it.

**Comment by theunknown86** [comment]
This is not xDrip related.  All Dexcom transmitters have internal consistency checks like regular firmware CRC verification & battery brownout detection. In the case of an error it will lock the transmitter & report a failed status. There is nothing xDrip can do to trigger this.

**Reply by spindulel** [reply]
Thank you for your answer.
